data.js

```diff
@@ -0,0 +1,176 @@
+  const {
+    name: projectName,
+    description: projectDescription,
+    featuredImage: projectFeaturedImage,
+    technologies: projectTechnologies,
+    stacks: projectStacks,
+  } = tonic;
+  const [canopy, backend, year] = projectStacks;
+  const [html, css, js, github, ruby, Bootstraps] = projectTechnologies;
+  document.querySelector('.tonic-title').textContent = projectName;
+  document.querySelector('.tonic-stack1').textContent = canopy.toUpperCase();
+  document.querySelector('.tonic-stack2').textContent = backend;
+  document.querySelector('.tonic-stack3').textContent = year;
+  document.querySelector('.tonic-description').textContent = projectDescription;
+  document.querySelector('.tonic-image').src = projectFeaturedImage;
+  document.querySelector('.tonic-html').textContent = html;
+  document.querySelector('.tonic-css').textContent = css;
+  document.querySelector('.tonic-javascript').textContent = js;
+  document.querySelector('.tonic-github').textContent = github;
+  document.querySelector('.tonic-ruby').textContent = ruby;
+  document.querySelector('.tonic-bootstraps').textContent = Bootstraps;
+
+  tonicModalContainer.classList.add('show');
+});
+
+closeTonicModal.addEventListener('click', () => {
+  tonicModalContainer.classList.remove('show');
+});
+
+// Multipost-Store Modal Details
+const openMultipostModal = document.querySelector('#open-multipost-modal');
+const multipostModalContainer = document.querySelector('.multipost-modal-container');
+const closeMultipostModal = document.querySelector('#multipost-modal-close');
+
+openMultipostModal.addEventListener('click', (e) => {
+  const multi = projects.find((project) => project.id === parseInt(e.currentTarget.dataset.id, 10));
+  const {
+    name: projectName,
+    description: projectDescription,
+    featuredImage: projectFeaturedImage,
+    technologies: projectTechnologies,
+    stacks: projectStacks,
+  } = multi;
+  const [canopy, backend, year] = projectStacks;
+  const [html, css, js, ruby] = projectTechnologies;
+  document.querySelector('.multipost-title').textContent = projectName;
+  document.querySelector('.multipost-stack1').textContent = canopy.toUpperCase();
+  document.querySelector('.multipost-stack2').textContent = backend;
+  document.querySelector('.multipost-stack3').textContent = year;
+  document.querySelector('.multipost-description').textContent = projectDescription;
+  document.querySelector('.multipost-image').src = projectFeaturedImage;
+  document.querySelector('.multipost-html').textContent = html;
+  document.querySelector('.multipost-css').textContent = css;
+  document.querySelector('.multipost-javascript').textContent = js;
+  document.querySelector('.multipost-ruby').textContent = ruby;
+
+  multipostModalContainer.classList.add('show');
+});
+
+closeMultipostModal.addEventListener('click', () => {
+  multipostModalContainer.classList.remove('show');
+});
+
```

Comment:
Great job working on the modal💪🏽. Please make sure the mobile popup follows the original Figma design for a better UI(User Interface).
Your design | Figma design
![screenshot (49)](https://user-images.githubusercontent.com/44624138/169423651-d30fb2be-a42f-41c2-b4e1-809624aaa12e.png) | ![screenshot (50)](https://user-images.githubusercontent.com/44624138/169423676-6b9bd86d-c987-426b-9850-e66b304e5852.png)